src/components/report/VariableFont.vue

```diff
@@ -9,7 +9,8 @@
 					autocorrect="off"
 					:style="
 						$filters.inlinestyle(variableStyles) +
-						` font-size: ${fontSize}px;`
+						` font-size: ${fontSize}px;` +
```

Comment:
Might make more sense to set a CSS Custom Property for the value and let CSS handle the rest instead of using a style attribute.